### DISCUSSION TITLE
How to use new menu with react router

### DISCUSSION THREAD

**Discussion by X-lem** [ant-design/ant-design#38047] [Q&A] [answered]
I'm seeing antd wants us to build menus like this now, but I'm not seeing how to integrate react router into this.

https://ant.design/components/menu/
```
// works when >=4.20.0, recommended ✅
const items = [
  { label: 'item 1', key: 'item-1' }, // remember to pass the key prop
  { label: 'item 2', key: 'item-2' }, // which is required
  {
    label: 'sub menu',
    key: 'submenu',
    children: [{ label: 'item 3', key: 'submenu-item-1' }],
  },
];
return <Menu items={items} />;
```

How do I handle this now that the `<Menu.Item>item 1</Menu.Item>` is deprecated? I'm not seeing any documentation for how I should go about this.

`<Link to={'/demo'}><span className='text'>Demo</span></Link>`

**Comment by abenhamdine** [comment, marked as answer]
you can pass a React Node with the label property, thus you should be able to replicate the legacy behaviour

**Reply by X-lem** [reply]
Ahhhh, much thanks!
